### ISSUE TITLE
SLIC superpixels not working on grayscale images.

### ISSUE THREAD

**Issue by marchese29** [scikit-image/scikit-image#1745]
Even when I set slic_zero to False, and with multi-channel set to false, I only get square superpixels with the attached image.  I can't find a set of parameters that make this work properly on grayscale images.
![stadium_bw](https://cloud.githubusercontent.com/assets/2624852/10339178/134fe482-6cd8-11e5-95dd-3a9aade2469f.jpg)

**Comment by emmanuelle** [comment]
If you get square superpixels, it means that ``compactness`` is too high. ``multichannel`` has no effect for a 2-D image. I tried for example
```
segmentation.slic(im, n_segments=200, compactness=0.1, enforce_connectivity=True))
```
and superpixels are not square, with boundaries that are preferentially found on edges between regions of different grayscales.

**Comment by jni** [comment]
We might want to say in the docstring that some parameter scans are best done initially on a log-scale, e.g. compactness=0.01, 0.1, 1, 10, 100, before trying more detailed values. What do you think, @emmanuelle? I say this because it's the second time people have had trouble with compactness.

**Comment by JDWarner** [comment]
:+1: when looking for insight on how to set unfamiliar parameters, the first place users will look is the docstring. They should be rewarded ;)

**Comment by jni** [comment]
Closed by #1750. Thanks for the report, @marchese29!
